packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -204,35 +208,6 @@ describe("DropdownComponent", () => {
 			hostFixture.detectChanges();
 		});
 
-		describe("on initialization", () => {
-			it("should set internal component properties", () => {
-				expect(hostFixture).toBeDefined();
-				expect(component).toBeDefined();
-				expect(component.logger).not.toBeNull();
-				expect(component.logger).toBeDefined();
-			});
-
-			it("should NOT have any inputs set", () => {
```

Comment:
Same remark: I think we should keep this unit test at least to be sure the properties are correctly initialized